### DISCUSSION TITLE
Resolution problem

### DISCUSSION THREAD

**Discussion by FunnyRabbit1234** [YimMenu/YimMenu#3246] [Q&A] [answered]
Please excuse me for the off-topic question, I just don’t know where to ask about it. I have a laptop with full-HD resolution and when I select the full-screen window and after each reboot of the game, the game does not open on the entire screen, but only on part of it. Has anyone encountered this? and how to fix it?
![screenshot](https://github.com/YimMenu/YimMenu/assets/172367170/c7e75e60-0913-4ce6-b6eb-795590dca728)

**Comment by D-Borkan** [comment, marked as answer]
you can try press ALT+ Enter both one time when be like your picture
